src/dash-table/components/PageNavigation/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { IPageNavigationProps } from 'dash-table/components/PageNavigation/props';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export default class PageNavigation extends Component<IPageNavigationProps> {
+
+    constructor(props: IPageNavigationProps) {
+        super(props);
+    }
+
+    loadNext = () => {
+        const { paginator } = this.props;
+        paginator.loadNext();
+    }
+
+    loadPrevious = () => {
+        const { paginator } = this.props;
+        paginator.loadPrevious();
+    }
+
+    loadFirst = () => {
+        const { paginator } = this.props;
+        paginator.loadFirst();
+    }
+
+    loadLast = () => {
+        const { paginator } = this.props;
+        paginator.loadLast();
+    }
+
+    goToPage = (page_number: string) => {
+        const { paginator } = this.props;
+
+        let page = parseInt(page_number, 10);
+
+        if (isNaN(page)) {
+            return;
+        }
+
+        paginator.goToPage(page);
+    }
+
+    render() {
+        const {
+            paginator,
+            page_current
+        } = this.props;
+
+        return (paginator.lastPage === 0 ? null : (
+            <div className='previous-next-container'>
+                <button
+                    className='first-page'
+                    onClick={this.loadFirst}
```

Comment:
No need for a wrapper function, just use it directly:
```suggestion
                    onClick={this.props.paginator.loadFirst}
```